python/BioSimSpace/Process/_gromacs.py

```diff
@@ -314,6 +320,8 @@ def _generate_config(self):
             config.append("nstlog = %d" % report_interval)      # Interval between writing to the log file.
             config.append("nstenergy = %d" % report_interval)   # Interval between writing to the energy file.
             config.append("nstxout = %d" % restart_interval)    # Interval between writing to the trajectory file.
+            if self._prev_cpt is not None:
```

Comment:
This would be `True` for anything other than a `NoneType`, i.e. the use could pass `False`, `'hello'`, etc.

Reply:
Now `self._prev_cpt` could be `NoneType` or valid file path.